crm_timesheet_analytic_partner/__openerp__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingenieria S.L. - Javier Iniesta
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "CRM Timesheet Analytic Partner",
+    "version": "8.0.1.0.0",
+    "images": [],
```

Comment:
It's only a warning, but it's intended for creating snapshots that are intesively used on new apps.odoo.com